templates/legal/terms-and-policies/cube-terms.html

```diff
@@ -11,14 +11,14 @@
 <div class="p-strip row">
```

Comment:
This isn't something you've changed but while you're here would you mind putting this whole page in a `<section class="p-strip>` and then one level down `<div class="row">` and then the `<div class="col-8">` for the rest of the content?